src/utilities/export-pdf-helper.ts

```diff
@@ -0,0 +1,242 @@
+                    ]
+                });
+
+                const obj = {
+                    sourceFilePath: gradeInstanceAttachments?.studentGradeInstance?.webworkQuestionPath ?? question.webworkQuestionPath,
+                    problemSeed: gradeInstanceAttachments?.randomSeed ?? gradeInstanceAttachments?.studentGradeInstance?.randomSeed ?? grade.randomSeed,
+                    formData: gradeInstanceAttachments?.submitted.form_data,
+                    showCorrectAnswers: true,
+                    answersSubmitted: 1,
+                    outputformat: rendererHelper.getOutputFormatForRole(Role.PROFESSOR),
+                    permissionLevel: rendererHelper.getPermissionForRole(Role.PROFESSOR),
+                    showSolutions: true,
+                };
+
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                let src: any = null;
+                try {
+                    src = await rendererHelper.getProblem(obj);
+                } catch (e) {
+                    logger.error(`Failed to load problem ${question.problemNumber} for ${grade.userId}.`);
+                }
+
+                studentGradeLookup[grade.userId].push({
+                    number: question.problemNumber,
+                    attachments: gradeInstanceAttachments?.studentGradeInstance?.problemAttachments?.map(attachment => ({
+                        url: urljoin(configurations.attachments.baseUrl, attachment.cloudFilename),
+                        name: attachment.userLocalFilename,
+                        time: attachment.updatedAt,
+                    })),
+                    effectiveScore: grade.effectiveScore * 100,
+                    legalScore: grade.legalScore * 100,
+                    srcdoc: src?.renderedHTML ?? 'Could not render this problem.'
+                });
+            })
+        );
+
+        await asyncForOwn(studentGradeLookup, async (studentGrades, userId) => {
+            const user = await User.findOne({
+                attributes: ['id', 'firstName', 'lastName'],
+                where: {
+                    id: userId,
+                    active: true,
+                }
+            });
+            
+            if (_.isNil(user)) {
+                logger.warn(`Could not find User ${userId}, probably deleted.`);
+                return;
+            }
+
+            const postObject: BulkPdfExport = {
+                topic: {
+                    id: mainData.id,
+                    name: mainData.name,
+                },
+                firstName: user.firstName,
+                lastName: user.lastName,
+                problems: studentGrades,
+                professorUUID: professorUUID
+            };
+            return await this.bulkExportAxios.post('export/', postObject);
+        });
+        
+        // Request zip
+        try {
+            return await this.bulkExportAxios.get(`export/?profUUID=${professorUUID}&topicId=${topic.id}`);
```

Comment:
Why are you manually doing the query parameters and not using the params option for axios?